tools/amd_build/build_amd.py

```diff
@@ -116,6 +116,21 @@ def is_hip_clang():
     except IOError:
         return False
 
+# TODO Remove once gloo submodule is recent enough to contain upstream fix.
```

Comment:
Is there already a PR to fix this in gloo?

Reply:
https://github.com/facebookincubator/gloo/pull/261